test/pageobjects/PassportUploadImageGuide.js

```diff
@@ -50,11 +51,23 @@ export default class PassportUploadImageGuide extends BasePage {
     // Input here cannot use the uploadInput() function above
     const input = this.$('.onfido-sdk-ui-CustomFileInput-input')
     const pathToTestFiles = '../resources/'
-    // This will detect local file, ref: https://www.browserstack.com/automate/node#enhancements-uploads-downloads
-    this.driver.setFileDetector(new remote.FileDetector())
-    const sendKeysToElement = input.sendKeys(
-      path.join(__dirname, pathToTestFiles + filename)
-    )
+    //the below line...if safari AND Local, ignore it...otherwise file uploads will fail.
+    if (browserName === 'safari' && isRemoteBrowser === false) {
```

Comment:
I noticed in the [`enableCameraAccessIfNecessary`](https://github.com/onfido/onfido-sdk-ui/blob/ccd5551b54274e8df910a9d31ba4ee7dd774ceb9/test/pageobjects/Camera.js#L47) function you are checking for both `browserName === 'safari' || browserName === 'Safari'` but the 2nd `browserName` case isn't required here also?

Reply:
Yes thats correct. The way Selenium reads the name is like this... 'safari' is local and 'Safari' is browserstack.